src/Validator.js

```diff
@@ -0,0 +1,28 @@
+import GameMessage  from "./GameMessage.js";
+class Validator {
+    validInput(input){
+        this.isNumber(input)
+        this.isValidLength(input)
+        this.isDuplicate(input)
+        return true
+    }
+
+    isNumber(input){
+        if (isNaN(input)) throw new Error(GameMessage.INVALID_INPUT_NOT_A_NUMBER)
+        return true
+    }
+    // 먼저 길이 체크 후 중복 검사
+    isValidLength(input) {
+        if (input.length !== 3) throw new Error(GameMessage.INVALID_INPUT_NOT_A_CORRECT_LENGTH);
+        return true
+    }
+
+    isDuplicate(input){
+        const set = new Set(input)
+        const newArr = [...set]
```

Comment:
1. 코드를 다음과 같이 고치면 더 줄어듭니다 ㅎㅎ
```suggestion
        const newArr = [...new Set(input)]
```
2. newArr이라는 상수명이 어떤 의미를 담고 있는 이름인가요? 의미가 없고, 단발성으로 사용되는 측면에서
이렇게 작성해도 될 것 같아요.
```js
isDuplicate(input) {
  if ([...new Set(input)].length !== 3) throw new Error(GameMessage.INVALID_INPUT_IS_DUPLICATED)
  return true;
}
```